src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In JavaScript, which Web API is used to insert a child node anywhere under the parent node?",
+    Answer: "insertBefore()",
+    Distractor1: "prepend()",
+    Distractor2: "append()",
+    Distractor3: "lookupPrefix()",
+    Explanation:
+      "The insertBefore() method insert a new child node before the reference node(The node we want to place before it) under the parent node and this method having two parameter: newNode and referenceNode",
```

Comment:
This is better. I think we can simplify this even more and if users want to have a more detailed explanation, then they can always click on the learn more link in the quiz. 

```suggestion
      "The insertBefore() method inserts a new child node anywhere under the parent node. This method has two parameters: newNode and referenceNode",
```